docs/docs/50-user-guide/60-reference-docs/70-annotations.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents certain annotations and labels that are applicable to
+Kargo resource types and other Kubernetes resource types (e.g. `Secret`s or
+`ServiceAccounts`) and affect how Kargo handles those resources. The subset
+documented here are those that the maintainers have judged most likely to be of
+use to advanced users who may, for instance, be looking to interact
+programmatically with Kargo.
+
+## Annotations
+
+| Key | Target Resource(s) | Possible Values | Description |
+|-----|--------------------|-----------------|-------------|
+| `kargo.akuity.io/refresh` | `Warehouse`, `Stage`, `Promotion` | A string that is unique or at least unlikely to repeat, such as a UUID or a timestamp of "now" | Triggers reconciliation of the resource when its value changes. |
+| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the current `Freight`. |
```

Comment:
Still need to know where to find a verification ID and how the JSON object is structured.

If it makes more sense to not mention the JSON option, that would probably be ok too.